convert2rhel/actions/system_checks/duplicate_packages.py

```diff
@@ -0,0 +1,50 @@
+# Copyright(C) 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import logging
+
+from convert2rhel import actions, utils
+
+
+logger = logging.getLogger(__name__)
+
+
+class DuplicatePackages(actions.Action):
+    id = "DUPLICATE_PACKAGES"
+
+    def run(self):
+        """Ensure that there are no duplicate system packages installed."""
+        super(DuplicatePackages, self).run()
+
+        logger.task("Prepare: Check if there are any duplicate installed packages on the system")
+        output, _ = utils.run_subprocess(["/usr/bin/package-cleanup", "--dupes", "--quiet"], print_output=False)
```

Comment:
Also, the package `package-cleanup` is part of the `dnf-utils` package. I just saw that in our specfile we have a comment saying we require dnf-utils because of yumdownloader. I think it would be nice if updated the comment there and specify this package name as well.

Reply:
I'll add that!

Reply:
Did we add this? Can be done in a different PR

Reply:
Yes I made the addition already @SpyTec